cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import socket
+import time
+import os
+
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Waits for database to be ready for connection"  # noqa: VNE003
+
+    def handle(self, *args):
+        port = int(os.environ["POSTGRES_PORT"])  # 5432
+
+        db_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+        print("Waiting for DB connection...")
+
+        while True:
+            try:
+                db_socket.connect(("db", port))
```

Comment:
The host name 'db' is hardcoded here. It would be better to take it from the environment variables just like the port.